.github/actions/count-words/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Count words
+description: In a PR, post a message with the difference of new sentences in each language
+inputs:
+  languages:
+    description: Languages to parse
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Checkout main branch to diff against
+      uses: actions/checkout@v4
+      with:
+        ref: main
+        path: main
+
+    - name: Count words
+      shell: bash
+      env:
+        LANGUAGES: ${{ inputs.languages }}
+      run: |
+        SUMMARY_FILE=`mktemp -p $PWD summary.XXXXX.txt`
+        cd main ;
+
+        IFS=',' read -ra LANG_ARRAY <<< "$LANGUAGES"
+        for lang in "${LANG_ARRAY[@]}"; do
+          echo -n "Counting previous sentences for language: " | tee -a ${SUMMARY_FILE}
+          python3 -m script.intentfest count_sentences --language "$lang" | grep -vE '^-' | tr -d ',' | sort -k2nr | tee -a ${SUMMARY_FILE}
```

Comment:
For a future PR, let's add an arg to `count_sentences` to avoid having to grep, tr and sort